### DISCUSSION TITLE
Busco Equipo !! 😬🥲

### DISCUSSION THREAD

**Discussion by MarcoAntonio159** [Launch-X-Latam/MisionFrontEnd#550] [Team Posts]
Hola,
Con todos, solo quiero indicarles que estoy en la busqueda de un equipo para el hackaton, mi correo es lx05663@innovaccion.mx, de antemano agradesco su ayuda.

Saludos.

**Comment by itsgaby** [comment]
Hola yo tambien estoy en busca de equipo, aun tienes espacio? lx05797@innovaccion.mx

**Comment by DagonNR** [comment]
Yo también busco equipo, ya están completos? lx15217@innovaccion.mx

**Comment by LaunchxUser** [comment]
Buen día.

Mismo caso, aun no cuento con equipo, ¿Hay espacio disponible?
mi correo es el siguiente: ***@***.***

Gracias.


________________________________
De: Daniel ***@***.***>
Enviado: viernes, 11 de noviembre de 2022 02:03 p. m.
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Busco Equipo !! 😬🥲 (#86)


Yo también busco equipo, ya están completos? ***@***.******@***.***>

**Comment by oscararcee** [comment]
Hola aun les falta integrante, yo no tengo equipo. 

Saludos
